action.yml

```diff
@@ -6,6 +6,10 @@ inputs:
   directory:
     description: "The directory to scan"
     required: true
+  arguments:
+      description: "Passes through options"
+      required: false
+      default: ""
```

Comment:
I don't believe it's necessary to specify the defaults (but I could be wrong).

Reply:
Specifying the default in the script works too. Made the change